pages/user/moderation.js

```diff
@@ -2207,45 +2207,58 @@ const UserModerationQueue = ({
   )
 }
 
-const RejectionModal = ({ id, profileIdToReject, rejectUser, signedNotes }) => {
+const RejectionModal = ({ id, profileToReject, rejectUser, signedNotes }) => {
   const [rejectionMessage, setRejectionMessage] = useState('')
   const selectRef = useRef(null)
 
+  const currentInstitutionName = profileToReject?.content?.history?.find(
+    (p) => !p.end || p.end >= new Date().getFullYear()
+  )?.institution?.name
+
   const instructionText =
     'Please go back to the sign up page, enter the same name and email, click the Resend Activation button and complete the missing data.'
```

Comment:
"click the Resend Activation button and follow the activation link to update your information'